pulpcore/pulpcore/app/serializers/task.py

```diff
@@ -85,12 +97,17 @@ class TaskSerializer(ModelSerializer):
         read_only=True
     )
 
+    created_resources = CreatedResourceSerializer(
```

Comment:
help_text please, I the docstring suggestion could work here too.

Reply:
👍